src/libomemo.mk

```diff
@@ -0,0 +1,33 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := libomemo
+$(PKG)_WEBSITE  := https://github.com/gkdr/libomemo
+$(PKG)_DESCR    := Implementation of OMEMO in C
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 0.4.1
+$(PKG)_CHECKSUM := af66fd2958dc5d6b23bc488b69e9f431bfb308d79bdd1b1c31de4575862c5142
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/gkdr/libomemo/archive/v$($(PKG)_VERSION).tar.gz
```

Comment:
After #1712, you can replace $(PKG)_FILE, $(PKG)_SUBDIR, $(PKG)_URL, and $(PKG)_UPDATE with:

$(PKG)_GH_CONF